kubernetes-pipeline/values.yaml

```diff
@@ -19,9 +19,18 @@ wso2Password: ""
 # Admin credentials of jenkins instance to be created
 jenkins:
```

Comment:
@Aaquiff I think it is better if we can add more detailed comments for the `values.yaml` of the pipeline.

Please create a separate issue if needed.

Reply:
Yes, created #49